packages/core/notifications/core-notifications-browser/tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+  "extends": "../../../../tsconfig.bazel.json",
+  "compilerOptions": {
+    "declaration": true,
+    "declarationMap": true,
+    "emitDeclarationOnly": true,
+    "outDir": "target_types",
+    "rootDir": "src",
+    "stripInternal": false,
+    "types": [
+      "jest",
+      "node",
+      "react"
```

Comment:
NIT we can remove the `react` type definition from this package, as it does not seem to be using any react types.